aptos-move/framework/aptos-stdlib/sources/type_info.move

```diff
@@ -1,12 +1,27 @@
 module aptos_std::type_info {
     use std::string;
+    use std::features;
+
+    //
+    // Error codes
+    //
+
```

Comment:
nit: Remove empty space. Here and below

Reply:
You mean line 8? That's intentional, since there might be other `///` comments below this "Error codes" comment block, and I want these to be separated.

Reply:
(There's no `///` comments now because `type_info` is not well-documented...)